src/software/ai/hl/stp/tactic/shoot_goal_tactic.h

```diff
@@ -123,7 +123,7 @@ class ShootGoalTactic : public Tactic
 
     // How far from the ball an enemy must be to be considered a danger that may steal the
     // ball
-    const double ENEMY_DANGER_DIST = 0.5 + ROBOT_MAX_RADIUS_METERS;
+    [[maybe_unused]] const double ENEMY_DANGER_DIST = 0.5 + ROBOT_MAX_RADIUS_METERS;
```

Comment:
what does this mean?

Reply:
Looks like it's unused. I'd prefer to just delete it in that case.

Reply:
yeah ill remove